src/Microsoft.Health.Fhir.CosmosDb.Initialization/Features/Storage/NullcollectionSetup.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Health.Fhir.CosmosDb.Core.Configs;
+using Polly;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.Features.Storage
+{
+    public class NullcollectionSetup : ICollectionSetup
```

Comment:
I think it makes sense to rename this class to "**Empty**CollectionSetup". 
It also makes sense to mark it as sealed.

Reply:
Let's also add a comment saying that this class does not execute any real initialization. 
It's used as a placeholder to external initialization.

Reply:
done